tests/components/orderentry/SortAndFilter.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import SortAndFilter from '../../../app/js/components/orderEntry/SortAndFilter';
+import sortAndFilterAction from '../../../app/js/actions/sortAndFilterAction';
+
+const store = mockStore({})
+const props = {
+  sortAndFilterAction: jest.fn()
+}
+const wrapper = shallow(<SortAndFilter store={store} />).dive();
+
+describe('sort and Filter component test-suite', () => {
+  it('renders properly', () => {
+    const selectElement = wrapper.find('select').at(0)
+    selectElement.simulate('change', { target: { value: 'testorder' }});
+    expect(wrapper.exists()).toBeTruthy();
+    expect(selectElement.exists()).toBeTruthy();
+  });
```

Comment:
I don't think this assertion: 
```
expect(selectElement.exists()).toBeTruthy();
```
is enough to test that the component `SortAndFilter` is rendering properly.
I suggest you generate a snapshot to confirm that the component is properly rendered.

Reply:
addressed